src/app/components/media/ReportDesigner/ReportDesignerComponent.js

```diff
@@ -271,5 +273,32 @@ ReportDesignerComponent.propTypes = {
   setFlashMessage: PropTypes.func.isRequired,
 };
 
-// TODO: createFragmentContainer
-export default withSetFlashMessage(ReportDesignerComponent);
+// Used in unit test
+// eslint-disable-next-line import/no-unused-modules
+export { ReportDesignerComponent };
+
+export default createFragmentContainer(withSetFlashMessage(ReportDesignerComponent), {
+  media: graphql`
+    fragment ReportDesignerComponent_media on ProjectMedia {
+      id
+      dbid
+      archived
+      permissions
+      last_status
+      dynamic_annotation_report_design {
+        id
+        dbid
+        data
+      }
+      team {
+        slug
+        get_language
+        get_languages
+        verification_statuses
```

Comment:
Isn't `...ReportDesignerForm_team` supposed to be here?

Reply:
:white_check_mark: